app/utilities/UserHelper.java

```diff
@@ -0,0 +1,148 @@
+package utilities;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import models.Address;
+import models.Token;
+import models.User;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.apache.commons.codec.binary.Base64;
+import org.apache.commons.codec.digest.DigestUtils;
+import org.apache.http.ParseException;
+import play.libs.Json;
+
+
+/**
+ * Created by test on 03/02/16.
+ */
+public class UserHelper {
+
+    private User user;
+
+    public UserHelper(JsonNode json){
+        System.out.println(">>>>>>json " + json);
```

Comment:
System.out should only be used for testing purpose. It should be replaced with Logger (ticket will be created for this).